### ISSUE TITLE
Network Diagnostics Tool - Accessing download links obtained from GET logs call returns 404 error

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#9296]
**Rancher versions:**
rancher/server:v1.6.3

**Docker version: (`docker version`,`docker info` preferred)**1.13.1

**Operating system and kernel: (`cat /etc/os-release`, `uname -r` preferred)**Ubuntu

**Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)**GCE

**Setup details: (single node rancher vs. HA rancher, internal DB vs. external DB)**Single node

**Environment Template: (Cattle/Kubernetes/Swarm/Mesos)**Cattle

**Steps to Reproduce:**
1. Deploy Network diagnostics catalog
2. Create logs using the below call:
```
soumyas-MBP:notes soumya$ curl -X POST http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs
{"actions":{},"downloadURL":"","id":"1499721000891963678","links":{"self":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/1499721000891963678"},"self":"1499721000891963678","state":"creating","type":"log"}
```
3. Get logs using : 
```
soumyas-MBP:notes soumya$ curl http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs
{"data":[{"actions":{},"downloadURL":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/rancher-logs-1499721000891963678.zip","id":"1499721000891963678","links":{"self":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/1499721000891963678"},"self":"1499721000891963678","state":"created","type":"log"},{"actions":{},"downloadURL":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/rancher-logs-1499719667364185965.zip","id":"1499719667364185965","links":{"self":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/1499719667364185965"},"self":"1499719667364185965","state":"created","type":"log"},{"actions":{},"downloadURL":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/rancher-logs-1499471302557385598.zip","id":"1499471302557385598","links":{"self":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/1499471302557385598"},"self":"1499471302557385598","state":"created","type":"log"},{"actions":{},"downloadURL":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/rancher-logs-1499704682282799514.zip","id":"1499704682282799514","links":{"self":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/1499704682282799514"},"self":"1499704682282799514","state":"created","type":"log"},{"actions":{},"downloadURL":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/rancher-logs-1499707278041946515.zip","id":"1499707278041946515","links":{"self":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/1499707278041946515"},"self":"1499707278041946515","state":"created","type":"log"},{"actions":{},"downloadURL":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/rancher-logs-1499707323681814087.zip","id":"1499707323681814087","links":{"self":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/1499707323681814087"},"self":"1499707323681814087","state":"created","type":"log"},{"actions":{},"downloadURL":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/rancher-logs-1499719689262516911.zip","id":"1499719689262516911","links":{"self":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/1499719689262516911"},"self":"1499719689262516911","state":"created","type":"log"},{"actions":{},"downloadURL":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/rancher-logs-1499720179640355553.zip","id":"1499720179640355553","links":{"self":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/1499720179640355553"},"self":"1499720179640355553","state":"created","type":"log"}],"links":{"self":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs"},"resourceType":"log","type":"collection"}
```
The download links are not accessible. Fails with the below error:
```
{
"actions": { },
"links": {
"self": "…/r/projects/1a5/network-diagnostics:8080/v1/logs/rancher-logs-1499720179640355553.zip",
},
"message": "log not found",
"status": 404,
"type": "error",
}
```

**Comment by leodotcloud** [comment]
Fixed by stack version: 0.0.2, image: rancher/network-diagnostics:v0.1.2

**Comment by soumyalj** [comment]
Tested with v0.0.2 version of the tool on rancher v1.6.8-rc1. The download links obtained from the GET calls are functional. The zip folders can be downloaded and extracted.
